starter-code/index.html

```diff
@@ -0,0 +1,284 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Bootstrap Revera</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+            integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://kit.fontawesome.com/59836ea697.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <main class='container-fluid'>
```

Comment:
Good use of `<main>...</main>`here.

I don't think that using `class='container-fluid'` on it is good because this class usually has elements with `class='row'` nested and then elements with `class='col-x'` (as you're doing from the `<header>`) soI think you should remove it.